### PR TITLE
testutil/compose: integrate p2p fuzz to compose

### PR DESCRIPTION
**Pull request by dB2510**
Integrates p2p fuzz to compose.

category: test
ticket: #2375

### CODE REVIEW

File: testutil/compose/fuzz/fuzz_test.go [hunk 1]
```diff
@@ -58,3 +58,35 @@ func TestBeaconFuzz(t *testing.T) {
...
+		t.Skip("Skipping p2p fuzz integration test")
+	}
+
+	dir, err := os.MkdirTemp("", "")
+	require.NoError(t, err)
+
+	conf := compose.NewDefaultConfig()
+	conf.DisableMonitoringPorts = true
+	conf.BuildLocal = true
+	conf.P2PFuzz = true
```
Comment (corverroos) [inline testutil/compose/fuzz/fuzz_test.go:73]: seems like all nodes are fuzzing so all duties will fail.

File: testutil/compose/run.go [hunk 2]
```diff
@@ -41,10 +41,16 @@ func Run(ctx context.Context, dir string, conf Config) (TmplData, error) {
 		nodes = append(nodes, n)
 	}
 
+	charonCmd := cmdRun
+	if conf.P2PFuzz {
+		nodes[0].EnvVars = append(nodes[0].EnvVars, kv{"p2p-fuzz", fmt.Sprintf(`"%v"`, conf.P2PFuzz)})
+		charonCmd = cmdUnsafeRun
+	}
```
Comment (dB2510) [inline testutil/compose/run.go:48]: @corverroos only node0 has `p2p-fuzz` enabled.

File: testutil/compose/fuzz/fuzz_test.go (hunk 1 above)
Comment (dB2510) [reply testutil/compose/fuzz/fuzz_test.go:73]: this config field is introduced to enable p2p fuzz in "one" of the nodes in compose cluster.

File: testutil/compose/run.go (hunk 2 above)
Comment (corverroos) [reply testutil/compose/run.go:48]: ah cool

**Review by corverroos — APPROVED** [review]

**Review by corverroos — APPROVED** [review]

**Pull request merged**